handlers/agentinfo/info.go

```diff
@@ -72,6 +74,7 @@ type agentStats struct {
 	LatencyMillis       *int64   `json:"lat,omitempty"`
 	PayloadBytes        *int     `json:"load,omitempty"`
 	StatusCode          *int     `json:"code,omitempty"`
+	ImdsFallbackSucceed *bool    `json:"ifs,omitempty"`
```

Comment:
Should this be an int instead? It'll need to be converted into an int at some point.

Reply:
Changed json output to int